lib/Report/SummaryReport.php

```diff
@@ -303,6 +303,7 @@ public function getSavedReportResults($json, $savedReport)
     /** @inheritDoc */
     public function getResults($filterCriteria, Request $request)
     {
+        // This function is called by both web ui and xtr,  XTR will have filters in $filterCriteria array and web in $request params
```

Comment:
Maybe we should handle this one step up? So whatever calls getResults makes the decision about which array of params to pass down?

Reply:
Agreed, attempt to achieve that in the next commit.